guides/05_custom-components/05_frontend.md

```diff
@@ -270,6 +270,15 @@ Svelte options:
 
 The `gradio.config.js` file should be placed in the root of your component's `frontend` directory. A default config file is created for you when you create a new component. But you can also create your own config file and use it to customize your component's build process.
 
+### Example for TailwindCSS
+
+If you want to use [TailwindCSS](https://tailwindcss.com) you can install it using version 4.0-alpha. 
```

Comment:
I think we should add a paragraph here just explaining that we can use vite plugins that allow you to customise the build process and maybe a link to the vite plugins docs.

Reply:
Done